runtime/queries/r/highlights.scm

```diff
@@ -0,0 +1,128 @@
+; highlights.scm
+
+
+; Literals
+
+(integer) @constant.numeric.integer
+
+(float) @constant.numeric.float
+
+(complex) @constant.numeric.integer
+
+(string) @string
+(string (escape_sequence) @constant.character.escape)
+
+(comment) @comment
+
+(identifier) @variable
```

Comment:
could you move this line down to just above the `(ERROR) @error` line?

This will have higher precedence than the rules below it (for helix: earlier stanzas have higher precedence) that target the `identifier` node. nvim-treesitter has the reverse precedence and it looks like that's where these queries originally came from.

Reply:
Ah I see, ok, that should be fixed by fa5baa3ea8f53fd4274f6e24211fae362d2fc83e.